src/coreclr/jit/instr.cpp

```diff
@@ -1485,6 +1485,9 @@ instruction CodeGen::ins_Move_Extend(var_types srcType, bool srcInReg)
 #elif defined(TARGET_ARM)
     assert(!varTypeIsSIMD(srcType));
     return INS_vmov;
+#elif defined(TARGET_LOONGARCH64) || defined(TARGET_RISCV64)
+    NYI("ins_Move_Extend"); // not used for LA64 and RISCV64.
+    return INS_invalid;
 #else
     NYI("ins_Move_Extend");
 #endif
```

Comment:
Fix the fallback case instead?

```suggestion
#else
    NYI("ins_Move_Extend");
    return INS_invalid;
#endif
```

Reply:
I don't update the `else`, I think this is value for adding a new architecture if they ignore this will be meeting a building error.

should I update this.